modules/ROOT/pages/reports.adoc

```diff
@@ -6,11 +6,13 @@ endif::[]
 *https://www.mulesoft.com/anypoint-pricing[This feature requires a Titanium subscription.]*
 
 == About Reports
-Reports allow you to view a list of all the CloudHub apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
+Reports allow you to view a list apps of running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
```

Comment:
Shouldn't it be "of apps" instead?

Reply:
Yes. My brain is not working. Thank you 😂

Reply:
```suggestion
Reports allow you to view a list of apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
```